src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordConstructor.cs

```diff
@@ -15,7 +15,12 @@ internal sealed class SynthesizedRecordConstructor : SourceConstructorSymbolBase
              DiagnosticBag diagnostics) :
              base(containingType, syntax.Identifier.GetLocation(), syntax, isIterator: false)
         {
-            this.MakeFlags(MethodKind.Constructor, containingType.IsAbstract ? DeclarationModifiers.Protected : DeclarationModifiers.Public, returnsVoid: true, isExtensionMethod: false);
+            this.MakeFlags(
+                MethodKind.Constructor,
+                containingType.IsAbstract ? DeclarationModifiers.Protected : DeclarationModifiers.Public,
+                returnsVoid: true,
+                isExtensionMethod: false,
+                isNullableEnabled: false); // IsNullableEnabled uses containing type instead.
```

Comment:
>isNullableEnabled: false); // IsNullableEnabled uses containing type instead. [](start = 16, length = 77)

Where the primary constructor declaration itself contributes to this value?

Reply:
Added to `AddSynthesizedRecordMembersIfNecessary()`.

---
In reply to: [546822298](https://github.com/dotnet/roslyn/pull/49953#discussion_r546822298) [](ancestors = 546822298)